docs/talks/scipy2019/submission.rst

```diff
@@ -0,0 +1,141 @@
+Zarr - scalable storage of tensor data for use in parallel and distributed computing
+====================================================================================
+
+SciPy 2019 submission.
+
+
+Short summary
+-------------
+
+(target: 100 words)
+
+Many scientific problems involve computing over large N-dimensional
+typed arrays of data, and reading or writing data is often the major
+bottleneck limiting speed or scalability. The Zarr project is
+developing a simple, scalable approach to storage of such data in a
+way that is compatible with a range of approaches to distributed and
+parallel computing. We describe the Zarr protocol and data storage
+format, and the current state of implementations for various
+programming languages including Python. We also describe current uses
+of Zarr in malaria genomics, the Human Cell Atlas, and the Pangeo
+project.
+
+
+Abstract
+--------
+
+(target: 500 words)
+
+Background
+~~~~~~~~~~
+
+Across a broad range of scientific disciplines, data are naturally
+represented and stored as N-dimensional typed arrays, also known as
+tensors. The volume of data being generated is outstripping our
+ability to analyse it, and scientific communities are looking for ways
+to leverage modern multi-core CPUs and distributed computing
+platforms, including cloud computing. Retrieval and storage of data is
+often the major bottleneck, and new approaches to data storage are
+needed to accelerate distributed computations and enable them to scale
+on a variety of platforms.
+
+Methods
+~~~~~~~
+
+We have designed a new storage format and protocol for tensor data
+[1_], and have released an open source Python implementation [2_,
+3_]. Our approach builds on data storage concepts from HDF5 [4_],
+particularly chunking and compression, and hierarchical organisation
+of datasets. Key design goals include: platform-independent and as
+simple as possible to implement in other programming languages;
```

Comment:
Might shorten this to just "simple implementation", "simple design spec", or similar.